data/json/items/comestibles/carnivore.json

```diff
@@ -209,13 +209,24 @@
     "fun": 0,
     "flags": [ "EATEN_HOT" ]
   },
+  {
+    "id": "pepperhuman",
+    "copy-from": "human_flesh",
+    "type": "COMESTIBLE",
+    "name": { "str": "cooked pepper creep" },
+    "description": "A cooked slice of some unpleasant person properly seasoned with black pepper and salt.  Tastes great.",
+    "proportional": { "price": 1.5 },
+    "parasites": 0,
+    "fun": 2,
+    "flags": [ "EATEN_HOT" ]
+  },
   {
     "id": "meat",
     "copy-from": "flesh",
     "type": "COMESTIBLE",
     "name": { "str": "chunk of meat", "str_pl": "chunks of meat" },
     "description": "A chunk of raw meat.  You could eat it as-is, but cooking it is better.",
-    "weight": "296 g",
+    "weight": "275 g",
```

Comment:
Why did this happen and how did it affect all the recipes using chunks of meat?

Reply:
Most fool uses a material `flesh`, which has density 1,1. This `meat` had incorrect weight, because overall density was happened to be a bit bigger. Changing the weight of this one piece of meat also changed the weight of all meat product, that was inherited from it (which seems to be 28 different items)
What do you mean "affect all the recipes using chunks of meat"? there was some bug because of this?

Reply:
Ah, probably to fix the density test, I've noticed the removal from known bad density list.

Reply:
Hey FYI my PR was based on 296g per chunk of meat so basically all cooking recipes involving chunks of meat are going to be a little off now. Did you have a plan to fix that?

Reply:
How i can do it?

Reply:
tbh? probably making meat 296g again. so the issue is that scrap of meat (30g) is 2u surface_heat while chunk of meat (300g) is 20u, but now that chunk of meat is 275u it is instead 18-19u of surface_heat across every instance of cooking raw chunks of meat in the game which would need to be adjusted individually.
Alternatively, we can leave it as is, and/or change the formula to say "20u: cook a chunk of meat" instead of "20u: cook 300g of meat"